### ISSUE TITLE
Feature request: Preview mode for cosmetic blocking

### ISSUE THREAD

**Issue by eljuno** [brave/brave-browser#1027]
## Description 

Originally requested here https://twitter.com/eljuno_/status/1037846658556473344?s=09

Brave should provide a preview option for cosmetic blocking (right-click > Brave). So users can preview which area that will be removed. 

It's a request, since uBO also have this feature.

## Steps to Reproduce

   1. Open any site.
   2. Right-click ads space > Brave > block element via selector.
   3. Preview.
   4. OK.

## Actual result:

The current option is only OK and Cancel

## Expected result:
Have preview option

## Brave version (about:brave info) 

Brave 0.55.1
Chromium: 70.0.3528.4

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? 
- Is the issue reproducible on the latest version of Chrome?

**Comment by Brave-Matt** [comment]
Additionally, it may also be useful to be able to view (in some form) what has _already_ been blocked by Shields on a particular web page.

**Comment by bsclifton** [comment]
Closing as `wontfix` - we have a new cosmetic filter in place. I created https://github.com/brave/brave-browser/issues/8914 to track removing this older version
